### PR TITLE
[Tempo] Adding Tempo support

### PR DESCRIPTION
**Pull request by jplayout**
Add Tempo support to the kubernetes example to fix the kubernetes part of the issue #1

### CODE REVIEW

File: kubernetes/helmfile.yaml [hunk 1]
```diff
@@ -10,6 +10,8 @@ repositories:
     url: https://helm.elastic.co
   - name: bitnami
     url: https://charts.bitnami.com/bitnami
+  - name: tempo
+    url: https://github.com/grafana/helm-charts/
```
Comment (jplayout) [inline kubernetes/helmfile.yaml:14]: we could also point to the correct helm chart directly and not to the whole grafana home page.

File: kubernetes/helmfile.yaml [hunk 2]
```diff
@@ -10,6 +10,8 @@ repositories:
     url: https://helm.elastic.co
   - name: bitnami
     url: https://charts.bitnami.com/bitnami
+  - name: tempo
```
Comment (dduportal) [inline kubernetes/helmfile.yaml:13]:
```suggestion
  - name: foo
```
Je préfère foo à tempo

File: kubernetes/helmfile.yaml (hunk 1 above)
Comment (dduportal) [inline kubernetes/helmfile.yaml:14]:
```suggestion
  - name: grafana
    url: https://grafana.github.io/helm-charts
```

Comment (dduportal) [reply kubernetes/helmfile.yaml:14]: Ref. https://github.com/dduportal/jenkins-examples/pull/3/files#r977880919

File: kubernetes/helmfile.yaml [hunk 3]
```diff
@@ -30,3 +32,10 @@ releases:
   #   namespace: jaeger
   #   values:
   #     - "values/jaeger.yaml"
+
+  # adding tempo support issue #1
+  - name: tempo
+    chart: grafana/tempo
+    namespace: tempo
+    values:
+      - "values/tempo.yaml"
```
Comment (dduportal) [inline kubernetes/helmfile.yaml:41]:
```suggestion
      - "values/tempo.yaml"

```

File: kubernetes/values/tempo.yaml [hunk 4]
```diff
@@ -0,0 +1,2 @@
+tempo:
+  searchEnabled: true
```
Comment (dduportal) [inline kubernetes/values/tempo.yaml:2]:
```suggestion
  searchEnabled: true

```

**Review by dduportal — APPROVED** [review]
LGTM, many thanks!

**Pull request merged**